src/object/cli_ec.c

```diff
@@ -179,24 +219,34 @@ ec_array_encode(struct ec_params *params, daos_obj_id_t oid, daos_iod_t *iod,
 	unsigned int	 k = oca->u.ec.e_k;
 	unsigned int	 p = oca->u.ec.e_p;
 	daos_recx_t     *this_recx = &iod->iod_recxs[recx_idx];
+	uint64_t	 ss = len * k;
```

Comment:
minor nit re: this function....I also try to pack stack variables on 8 byte boundaries.   The compiler may or may not rearrange them but why make it work harder.